packages/frontend/src/components/NavBar/MenuList.tsx

```diff
@@ -1,6 +1,7 @@
 import { ExpandLess, ExpandMore } from '@mui/icons-material';
 import { ListItem, ListItemButton, ListItemIcon, ListItemText } from '@mui/material';
-import { useContext } from 'react';
+import React, { useContext } from 'react';
```

Comment:
Maybe import `{ Fragment }`? Not sure if that helps with any bundling optimisation or not.